README.md

```diff
@@ -23,12 +23,14 @@ To use SwiftFrame, you need to pass it a configuration file (which is a plain JS
 * `fontFile`: a path to a font file
 * `format`: the output format of the screenshots, can be `png`, `jpeg` or `jpg`
 * `textColor`: a RGB color in Hex format (e.g. `#FFF`) to use for titles
-* `outputWholeImage`: a boolean telling the application whether or not to also output the whole image instead of just the sliced up screenshots
-* `clearDirectories`: a boolean telling the application whether or not to clear the specified output directories before writing new files to it. This prevents random screenshots from being used in case you update your template file to include one less screenshot for example
+* `outputWholeImage`: **optional (default: false)** a boolean telling the application whether or not to also output the whole image instead of just the sliced up screenshots
+* `clearDirectories`: **optional (default: true)** a boolean telling the application whether or not to clear the specified output directories before writing new files to it. This prevents random screenshots from being used in case you update your template file to include one less screenshot for example
+* `skippedLocales`: an array of locale identifiers (strings) which will be skipped during rendering
```

Comment:
I wonder if we'll need the other case (don't skip these locales) as well? In our example we'll want to treat RU and anything else differently so one config would have to skip RU and the other _everything else_ which would be quite a long list (that would require updating if new locales are added).

Reply:
Not sure if maybe we could use a regex?

`en|ru` would allow only `en` and `ru`.

`(?!:en|au)` would allow anything apart from `en` and `ru`.

Reply:
That's a very good idea!